apps/settings/elements/full_developer_mode.html

```diff
@@ -0,0 +1,39 @@
+<element name="full-developer-mode" extends="section">
+  <template>
+    <div class="full-dev-mode-warning" hidden>
+      <gaia-confirm>
+        <h1 data-l10n-id="enable-full-dev-mode-first-warning-title"></h1>
+        <p>
+          <span data-l10n-id="enable-full-dev-mode-first-warning-msg-1"></span>
+          <br/>
+          <br/>
+          <span data-l10n-id="enable-full-dev-mode-first-warning-msg-2"></span>
+          <br/>
+          <br/>
```

Comment:
Let's not use `<br/>` here, how about using specific CSS rules to make do the same thing ? For me, `<br/>` is too general to be used.